src/settings.rs

```diff
@@ -46,6 +47,7 @@ mod tests {
 ---
 example:
   url: file:///tmp/namespace-validate-policy.wasm
+  mutating: false
```

Comment:
If we move to an optional configuration, that defaults to false, then I would allow for the configuration entry to be missing, for backwards compatibility. I would then remove all the `mutating: false` on the tests, and add just a new test with it set to true.